src/datatypes/interval.coffee

```diff
@@ -313,6 +313,7 @@ module.exports.Interval = class Interval
       Math.round(diff * Math.pow(10, 8)) / Math.pow(10, 8)
 
   size: (pointSize) ->
+    pointSize = @getPointSize()
```

Comment:
You're still accepting `pointSize` as an argument above but are immediately overwriting it.  I expect that's not the intent.

Reply:
Good catch--will remove